lib/README.md

```diff
@@ -0,0 +1,75 @@
+# Health Cards
```

Comment:
I think we should include the existing readme rather than creating a new one.

Reply:
I tried to make a one-for-all README in the repo root

Reply:
Is this file going to be removed now?

Reply:
removed